src/NuGet.Core/NuGet.Common/NuGet.Common.csproj

```
@@ -1,40 +1,42 @@
-<Project>
+﻿<Project>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'README.md'))\build\common.props" />
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
 
   <PropertyGroup>
     <Description>Common utilities and interfaces for all NuGet libraries.</Description>
-    <TargetFrameworks>$(TargetFrameworksLibrary)</TargetFrameworks>
+    <TargetFrameworks>$(TargetFrameworksLibrary);net45</TargetFrameworks>
```

We had a problem with source-build with hardcoded netfx TFM's in the past. To avoid the same problem in the future, avoid adding the net45 TFM in source-build mode:

```suggestion
    <TargetFrameworks>$(TargetFrameworksLibrary)</TargetFrameworks>
    <TargetFrameworks Condition="'$(IsBuildOnlyXPLATProjects)' != 'true'">$(TargetFrameworks);net45</TargetFrameworks>
```

---

fixed in next iteration.